test/jit/fixtures_srcs/generate_models.py

```diff
@@ -0,0 +1,178 @@
+import torch
+from fixtures_src import TestVersionedDivTensorExampleV4
+from pathlib import Path
+from typing import Set
+import io
+import zipfile
+import logging
+import sys
+from torch.jit.mobile import _load_for_lite_interpreter, _export_operator_list
+
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+"""
+This file is used to generate model for test operator change. Please refer to
+https://github.com/pytorch/rfcs/blob/master/RFC-0017-PyTorch-Operator-Versioning.md for more details.
+
+A systematic workflow to change operator is needed to ensure
+Backwards Compatibility (BC) / Forwards Compatibility (FC) for operator changes. For BC-breaking operator change,
+an upgrader is needed. Here is the flow to properly land a BC-breaking operator change.
+
+1. Write an upgrader in caffe2/torch/jit/operator_upgraders.py file. The softly enforced naming format
```

Comment:
Can this wait to be landed until I land the change where we move everything to cpp?

Reply:
Yeah of course.